internal/pkg/node/constructors.go

```diff
@@ -27,7 +27,7 @@ defaultnode:
   system overlay:
   - wwinit
   kernel:
-    args: quiet crashkernel=no vga=791 net.naming-scheme=v238
+    args: quiet crashkernel=no net.ifnames=1
```

Comment:
I'm in favor of removing `vga=791` and `net.naming-scheme=v238`; but isn't `net.ifnames=1` equivalent to just not specifying the parameter? If so, I suggest just removing these (and maybe the _other_ default kernel arguments, as well. Why does it need to default to `quiet` and `crashkernel=no`?

In any case, though, I think that this is too large a change for v4.5.x; so I'm moving it to v4.6.0.